plugin/src/main/kotlin/com/varabyte/kobweb/intellij/project/KobwebProject.kt

```diff
@@ -2,44 +2,20 @@ package com.varabyte.kobweb.intellij.project
 
 import com.intellij.openapi.module.Module
 import com.intellij.openapi.vfs.VirtualFile
-import com.varabyte.kobweb.intellij.util.kobweb.isInKobwebReadContext
-import com.varabyte.kobweb.intellij.util.kobweb.isInKobwebWriteContext
+import com.varabyte.kobweb.intellij.model.KobwebProjectType
 
 /**
  * A Kobweb Project is a module that has applied one of the Kobweb Gradle Plugins.
  *
  * It can either be a local project inside the user's workspace or an external dependency (e.g. a maven artifact).
+ *
+ * @property name A human-readable name for this project.
  */
 class KobwebProject(
     val name: String,
-    val type: Type,
+    val type: KobwebProjectType,
```

Comment:
https://testing.googleblog.com/2017/10/code-health-identifiernamingpostforworl.html

In this case, "type" is understood from the containing class context.

Reply:
I suggested this, because i took issue with the readability of this:
```kt
return isInKobwebContext { it.type in limitTo && it.source is KobwebProject.Source.Local }
```
But thinking about it, the proper way to solve this is probably renaming `it` to `project`

Reply:
Thanks for calling it out as confusing. Done.